packages/router/src/interfaces/Indexable.ts

```diff
@@ -0,0 +1,2 @@
+export type Indexable = { [key: string]: any };
+export type Params = Indexable;
```

Comment:
Why not reuse this from `grpc-sdk`?
(If there's an actual reason for this, export types from `../index.ts` and add new line at EOF.